File: programmers/level1/032.py
# 32. 이상한 문자 만들기
def solution(s):
    answer = ''
    j = 0
    for i in s:
        if i == ' ':
            answer+=i
            j=0
            continue
        elif j%2==1:
            answer+=i.lower()
        else :
            answer+=i.upper()
        j+=1
    return answer

File: programmers/level2/032.java
// 32. 더 맵게
import java.util.*;
class Solution {
    public int solution(int[] scoville, int K) {
        int answer = 0;
        PriorityQueue<Integer> pq = new PriorityQueue<>();
        
        for(int sco : scoville){
            pq.add(sco);
        }
        
        while(pq.peek()<K){
            try{
            int first = pq.poll();
            int second = pq.poll();
            pq.add(first + second*2);
            answer+=1;
            }catch(Exception e){
                answer=-1;
                break;
            }
        }
        return answer;
    }
}